src/app/src/components/QuizHint.js

```diff
@@ -1,23 +1,49 @@
 import React from 'react';
 import { string } from 'prop-types';
-import styled from 'styled-components';
+import styled, { keyframes } from 'styled-components';
 import { Box } from 'rebass';
 import { Text } from './custom-styled-components';
 import { themeGet } from 'styled-system';
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
+const slideIn = keyframes`
+    0% {
+        transform: translateX(100%) scale(1);
+    }
+
+    80% {
+        transform: translateX(80%) scale(1.15);
+    }
+  
+    100% {
+        transform: translateX(0) scale(1);
+    }
+`;
+
 const StyledQuizHint = styled(Box)`
-    background: ${themeGet('colors.greens.3')};
+    background: ${themeGet('colors.greens.2')};
     text-align: center;
     width: fit-content;
-    margin-left: auto;
+    position: absolute;
+    right: 0;
+    bottom: ${themeGet('space.comfortable')};
+    transform: translateX(100%);
+    border-radius: 10px 0 0 10px;
+    padding: ${themeGet('space.compact')};
+    animation: ${slideIn} 0.15s cubic-bezier(0.33, 0.1, 0.6, 0.84) 0.3s forwards;
```

Comment:
Is it possible to hide the browser scrollbars when the hint is offscreen, or possibly ignore the offscreen elements? There's a flash of scrollbars, vert and horiz, when the hint animates in.

Reply:
Oof, glad you noticed. I'll check and see—I imagine so.